README.md

```diff
@@ -53,7 +53,7 @@ python ./download_embeddings_model.py \
 ```
 python ./scripts/generate_embeddings_openstack.py \
         -o ./vector_db/ \
-        -of openstack-docs-plaintext/ \
+        -f openstack-docs-plaintext/ \
```

Comment:
:+1: I thought at first this might be an accidental reversion, but it checks out. :)